script/setup/SetupToyOrchestratorScript.s.sol

```diff
@@ -141,8 +141,8 @@ contract SetupToyOrchestratorScript is Test, DeploymentScript {
         // Now we need to find the MilestoneManager. ModuleManagerBase_v1 has a function called `listModules` that returns a list of
         // active modules, let's use that to get the address of the MilestoneManager.
 
-        // TODO: Ideally this would be substituted by a check that that all mandatory modules implement their corresponding interfaces + the same for MilestoneManager
-
+        //::@todo: Ideally this would be substituted by a check that that all mandatory modules implement their corresponding interfaces + the same for MilestoneManager
+        // @0xNuggan are these Contracts still up to date? Can we maybe remove them?
```

Comment:
The contract needs to be updated, but we shouldn't remove it since it's still in use when deploying to testnet. Created issue and assigned myself

Reply:
So if you created an issue for it can I remove the comment then?

Reply:
yep